IFComp/lib/IFComp/Schema/Result/Entry.pm

```diff
@@ -1004,12 +1004,10 @@ sub update_content_directory {
         $self->clear_play_file;
     }
     elsif ( $self->platform eq 'parchment' ) {
-        $self->_mangle_parchment_head;
+        $self->_enable_recording('parchment_options');
     }
     elsif ( $self->platform eq 'quixe' ) {
-        $self->_make_js_file( $self->inform_game_file,
-            $self->inform_game_js_file );
-        $self->_mangle_quixe_head;
+        $self->_enable_recording('game_options');
     }
```

Comment:
This is choosing which variable to insert based on whether it's a "parchment" or a "quixe" platform, but the labels for that on the entry form are "Glulx, using Inform's "Release with a website" feature" or "Z-code, using Inform's "Release with a website" feature" - ie, based on the virtual machine format, and not which interpreter template was used.

You could change the labels on the entry form, but then you'd be relying on the author to select the right option, and it's very easy to imagine someone selecting "Quixe" because they'd think "my game is Glulx and that means Quixe" even though they actually used the new Parchment template. Instead I'd recommend just looking for "game_options" or "parchment_options" in the play.html source code. We're wanting to reduce magic, but I think that's a very safe heuristic.